pootle/apps/pootle_misc/checks.py

```diff
@@ -1106,6 +1104,31 @@ def get_qualitycheck_schema(path_obj=None):
     return result
 
 
+def get_qualitycheck_list(path_obj):
+    """
+    Returns list of checks sorted in alphabetical order
+    but having critical checks first.
+    """
+    result = []
+    checks = get_qualitychecks()
+
+    for check, cat in checks.items():
+        result.append({
+            'code': check,
+            'is_critical': cat == Category.CRITICAL,
+            'title': u"%s" % check_names.get(check, check),
```

Comment:
Locally tests work but I can't recall when we hit related errors and why. So I'd prefer to leave it as it is within this PR and to clean it up later.